App/App_macOS.swift

```diff
@@ -185,7 +186,18 @@ struct RootView: View {
             navigation.currentItem = .reading
             browser.load(url: url)
         }
```

Comment:
There are 2 ways to close the macOS application:
- CMD+Q  - this will terminate the application
- by closing each and every tab - clicking on X of the tab or pressing CMD+W for each tab/window.
The difference here is that if the user closes a tab, we should delete the data related to it, if we quit the app we should not delete tab data.